modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} Ingress Controller for dual-stack support
+
+When you install a {product-title} cluster on a IPv4/IPv6 dual-stack enabled bare metal infrastructure, your cluster to is externally reachable by {product-title} routes.
+
+The Ingress Controller automatically serves services that have both IPv4 and IPv6 endpoints, but you can configure the Ingress Controller for single-stack or dual-stack services.
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on bare metal.
+* You installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+To have the Ingress Controller serve traffic over IPv4/IPv6 to a workload, you can create or modify a service by setting the `ipFamilies` and `ipFamilyPolicy` fields.
+
+. Create service by entering the following command:
```

Comment:
"Create a service..."?

Also, the `oc get service` that follows retrieves a service rather than adds one.